frontend/packages/helm-plugin/src/components/list-page/HelmReleaseListHeader.tsx

```diff
@@ -3,13 +3,13 @@ import { sortable } from '@patternfly/react-table';
 import { Kebab } from '@console/internal/components/utils';
 
 export const tableColumnClasses = {
-  name: 'col-lg-2 col-md-4 col-sm-4 col-xs-4',
-  revision: 'col-lg-2 col-md-3 col-sm-3 col-xs-3',
-  updated: 'col-lg-2 col-md-3 col-sm-5 col-xs-5',
-  status: 'col-lg-1 col-md-2 hidden-sm hidden-xs',
-  chartName: 'col-lg-2 hidden-md hidden-sm hidden-xs',
-  chartVersion: 'col-lg-2 hidden-md hidden-sm hidden-xs',
-  appVersion: 'col-lg-1 hidden-md hidden-sm hidden-xs',
+  name: '',
+  revision: '',
+  updated: '',
+  status: 'pf-m-hidden pf-m-visible-on-lg',
+  chartName: 'pf-m-hidden pf-m-visible-on-xl',
+  chartVersion: 'pf-m-hidden pf-m-visible-on-xl',
+  appVersion: 'pf-m-hidden pf-m-visible-on-xl pf-u-w-8-on-xl',
```

Comment:
Another similar change here...

Cluster:
![Screen Shot 2021-05-25 at 3 27 33 PM](https://user-images.githubusercontent.com/8126518/119573472-51c20500-bd82-11eb-97bb-41e588bced7e.png)

PR:
![Screen Shot 2021-05-25 at 3 27 26 PM](https://user-images.githubusercontent.com/8126518/119573468-51296e80-bd82-11eb-92d0-18e1590611e2.png)

Reply:
Similar logic here, let me know what must be done?

Reply:
https://coreos.slack.com/archives/CHGU4P8UU/p1622035319164400